src/sources/hydra.rs

```diff
@@ -73,14 +102,44 @@ where
 }
 
 
+// Example JSON:
+// {
+//     "headId":"84e657e3dd5241caac75b749195f78684023583736cc08b2896290ab",
+//     "seq":6,
+//     "tag":"HeadIsOpen",
+//     "timestamp":"2024-10-07T11:45:12.219902045Z",
+//     "utxo": {
+//         "060d5256c6335978f59c8434dfec2f814276983c41949ffc150fa9a0d0cd5f6f#0": {
+//             "address":"addr_test1vqx5tu4nzz5cuanvac4t9an4djghrx7hkdvjnnhstqm9kegvm6g6c",
+//             "datum":null,
+//             "datumhash":null,
+//             "inlineDatum":null,
+//             "referenceScript":null,
+//             "value":{"lovelace":25000000}},
+//         "9dce218cadb86ec5e26e9bdddab614efa722f1d72f8b98eade4b5ce52bb2eb8a#0": {
+//             "address":"addr_test1vp0yug22dtwaxdcjdvaxr74dthlpunc57cm639578gz7algset3fh",
+//             "datum":null,
+//             "datumhash":null,
+//             "inlineDatum":null,
+//             "referenceScript":null,
+//             "value":{"lovelace":50000000}},
+//         "d683dda48161cef8015c40c55a51569c85631d294dc5e4ce39a29a54939ca6f3#0": {
+//             "address":"addr_test1vp5cxztpc6hep9ds7fjgmle3l225tk8ske3rmwr9adu0m6qchmx5z",
+//             "datum":null,
+//             "datumhash":null,
+//             "inlineDatum":null,
+//             "referenceScript":null,
+//             "value":{"lovelace":100000000}}
+//     }
+// }
```

Comment:
I think these could/should really be inline unit tests, but haven't tried to do it yet